### PR TITLE
Odds and ends

### PR DESCRIPTION
**Pull request by amolenaar**
### PR Type
What kind of change does this PR introduce?

- [ ] Bug fix
- [ ] Feature
- [x] Chore (refactoring, formatting, local variables, other cleanup)
- [ ] Documentation content changes

### What is the current behavior?

Just a few small left-over fixes.

### What is the new behavior?

- [x] Fix preferences width
- [x]  Fix show-editors button style
- [x] Add styling for Association, and Connector item flow
- [x] Add styling for Association ends
- [x] CSS styling for Lifeline lifetime
- [x] Add CSS styling for diagram pentagon (top left marker)
- [x] Interaction text should be left aligned (`interaction > pentagon`)
- [x] Fix issue with LifeLine when upper handles are moved (lifetime line is out of place)
- [x] Preferences pane no longer uses extra width.
 
### Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: gaphor/UML/classes/association.py [hunk 1]
```diff
@@ -420,24 +404,25 @@ def max_text_size(size1, size2):
 
         if abs_rc > 6:
             # horizontal line
-            name_dy = -ofs * 2 - name_h
-            mult_dy = ofs * 2
+            name_dy = -padding_top - padding_bottom - name_h
```
Comment (danyeaw) [inline gaphor/UML/classes/association.py:407]: This is much more readable :+1:

**Review by danyeaw — APPROVED** [review]
Thanks @amolenaar, just one nitpick :smile:

File: gaphor/UML/classes/association.py (hunk 1 above)
Comment (amolenaar) [reply gaphor/UML/classes/association.py:405]: And you can change the distance from CSS (although it doesn't take into account the angle of the line end.

File: gaphor/UML/classes/association.py [hunk 2]
```diff
@@ -180,10 +185,7 @@ def on_association_end_value(self, event=None):
                 self.draw_head = draw_default_head
             self.draw_tail = draw_default_tail
 
-        self.request_update()
-
-    def update(self, context):
-        self.update_ends()
+        # self.request_update()
```
[the remark this answers is not in the document]
Comment (amolenaar) [reply gaphor/UML/classes/association.py]: Fixed

**Pull request merged**